src/composables/common/useLoader.ts

```diff
@@ -0,0 +1,28 @@
+/*
+Composable function for loader component that is displayed whenever axios async process takes place
+Global loader is used in App.vue where show a full route loader - <loader-component v-show="globalLoader"></loader-component>
+While, isLoading can be used inside an individual component - <loader-component v-if="isLoading"></loader-component>
+<load>
+*/
+
+import { createNamespacedHelpers } from 'vuex-composition-helpers'
+import { ref } from '@vue/composition-api'
+
+const loadingStatusModule = createNamespacedHelpers('loadingStatus') // specific module name
+const { useState } = loadingStatusModule
+
+export function useLoader () {
+  const isLoading = ref<boolean>(false)
+  // vuex state and mutations
+  const { globalLoader } = useState(['globalLoader'])
+
+  function toggleLoading (): void {
```

Comment:
better to have one more function which will accept loading state and change it accordingly

Reply:
loadingstate is a state. Therefore, there is mutation that would change that value

Reply:
![image](https://user-images.githubusercontent.com/78374865/127713237-607a1dc7-c941-4fa1-8e84-214389dbcd35.png)